typescript/processors/readTypeScriptModules.js

```diff
@@ -140,9 +156,65 @@ module.exports = function readTypeScriptModules(tsParser, modules, getFileInfo,
           }
         });
       });
+
+      /** Resolves the inherited docs and stores them in the export doc. */
+      function resolveInheritedDocs(exportSymbol, exportDoc) {
+        var inheritedSymbols = resolveInheritedSymbols(exportSymbol, typeChecker);
+        var inheritedDocs = inheritedSymbols.map(function(symbol) {
+          var relatedDoc = docs.find(function(doc) { return doc.exportSymbol === symbol; });
```

Comment:
That's true. Should we use some faster search algorithms? Or do you have any better ideas? I thought about creating a Map of Symbols with their associated docs, but that would then fill-up memory..